indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -74,6 +74,26 @@
             $reviewBox.find('.js-edit-review').trigger('click');
             $('body, html').animate({scrollTop: $reviewBox.offset().top}, 'fast');
             $reviewBox.find('.i-timeline-item-box').effect('highlight', {color: Palette.highlight}, 'slow');
+        }).on('click', '.js-toggle-details', function(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+
+            function toggleDetails($trigger) {
```

Comment:
This particular case is not so generic since it's the implementation of an accordion, meaning that you cannot have more than 1 item open. Maybe this could be another data attribute...